pytensor/tensor/nlinalg.py

```diff
@@ -603,7 +616,7 @@ def svd(a, full_matrices: bool = True, compute_uv: bool = True):
     U, V,  D : matrices
 
     """
-    return SVD(full_matrices, compute_uv)(a)
+    return Blockwise(SVD(full_matrices, compute_uv))(a)
```

Comment:
You can define `gufunc_signature` in the SVD Op, so that python perform will just use the already batched `numpy` method and bypass the useless `vectorize`

Reply:
Nevermind, this has kwarg arguments